CUSTOM_CELLS.md

```diff
@@ -0,0 +1,76 @@
+# MessageKit Custom Cell Guide(s)
+
+- [How can I add a custom cell?](#how-can-i-add-a-custom-cell)
+
+## How can I add a custom cell?
+
+**Note:** If you choose to use the `.custom` kind you are responsible for all of the cell's layout. Any `UICollectionViewCell` can be returned for custom cells which means any of the styling you provide from the `MessageDisplayDelegate` will not effect your custom cell. Even if you subclass your cell from `MessageContentCell`.
+
+Creating a custom cell involves four parts. First, you will need to create your custom cell so that it inherits from `UICollectionViewCell`. Second, you will need to set the size of your custom cell by creating a class that inherits from `MessageSizeCalculator`. Third, you will need to add your custom message size calculator to a class you make that inherits from `MessagesCollectionViewFlowLayout`. Fourth, you will need to register your custom cell and reference your custom collection view flow layout.
```

Comment:
Maybe change the formatting to be easier to digest? Something like:

Creating a custom cell involves four parts:

1. Subclass `UICollectionViewCell` (or another UICollectionViewCell subclass).
2. Subclass `MessageSizeCalculator` if you want your cell to have the default MessageKit layout design. Subclass `CellSizeCalculator` if you want to further customize your own cell design. The implementation of this class will allow your custom cell to automatically size itself within the messagesCollectionView.
3. Subclass `MessagesCollectionViewFlowLayout`, and use the custom message size calculator from #2 in the implementation.
4. Register your custom cell and reference your custom collection view flow layout.

Reply:
Updated documentation to reflect these changes